retrofit/src/test/kotlin/com/badoo/reaktive/retrofit/StringConverterFactory.kt

```diff
@@ -0,0 +1,28 @@
+package com.badoo.reaktive.retrofit
+
+import okhttp3.MediaType
+import okhttp3.RequestBody
+import okhttp3.ResponseBody
+import retrofit2.Converter
+import retrofit2.Retrofit
+import java.lang.reflect.Type
+
+internal class StringConverterFactory : Converter.Factory() {
+
+    override fun responseBodyConverter(
+            type: Type,
+            annotations: Array<Annotation>,
+            retrofit: Retrofit
+    ): Converter<ResponseBody, String> {
```

Comment:
Single return function.
Convert to 
```kotlin
): Converter<ResponseBody, String> =
    Converter { responseBody -> responseBody.string() }
```

Reply:
Also, isn't it easier just to add dependency on `converter-scalars`?

Reply:
I didn't known about this library. Thx for information.